meson.build

```diff
@@ -135,6 +141,37 @@ if fontconfig
     font_providers += ['Fontconfig']
 endif
 
+threads_opt = get_option('threads')
+if not threads_opt.disabled()
+    thread_dep = dependency('threads')
+    if thread_dep.found()
```

Comment:
Note that `dependency('threads').found()` is currently [always true](https://github.com/mesonbuild/meson/blob/7ec1fc507ea6f80e819ac46b56125b955073de7f/mesonbuild/dependencies/misc.py#L149), even if the compiler does not actually support the `-pthread` argument (see also https://github.com/mesonbuild/meson/issues/7775).
Of course, it's always possible for this to change in the future, but I wanted to point this out in case you weren't aware.

Reply:
Good to know, though I'm not sure if there's much to be done about it at the moment.